code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -719,6 +719,14 @@
 	construction_time = 50
 	category = list("Misc")
 
+/datum/design/mecha_tracking_ai_control
+	name = "AI Control Beacon"
+	id = "mecha_tracking_ai_control"
+	build_path = /obj/item/mecha_parts/mecha_tracking/ai_control
+	materials = list(MAT_METAL = 1000, MAT_GLASS = 500, MAT_SILVER = 200)
+	construction_time = 50
+	category = list("Misc")
+
```

Comment:
You need to add `build_type = MECHFAB` in order to make it show up in the fabricator!